acceptance-test/settings.gradle

```diff
@@ -1 +0,0 @@
-rootProject.name = 'acceptance-test'
```

Comment:
I was not sure about that change, but the code seemed strange. I removed all of the settings.gradle files since there was already one in the root, and the overall speed of the project made it redundant to have every sub project on it's own. Perhaps it is wrong. We'll see.